sda-mq/templates/tests.yaml

```diff
@@ -0,0 +1,84 @@
+{{- define "mqfullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  "{{ .Release.Name }}-test-certs"
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+type: Opaque
+data:
+  {{- (.Files.Glob "files/{tester,CA,ca}*").AsSecrets | nindent 2 }}
+
+---
+
```

Comment:
extra space ? and maybe would be good to split into secret and  like we do for the rest of the YAMLs

the same comment for the other tests YAMLs